recipes/trinity/meta.yaml

```diff
@@ -13,8 +13,8 @@ source:
     - makefile.patch
 
 build:
-  number: 1
   skip: True  # [osx]
+  number: 1
```

Comment:
was that supposed to be build:2?

Reply:
No, it was blacklisted during the last build increment due to failing, so build 1 never existed as a package.